sdk/python/cmd/pulumi-language-python/main.go

```diff
@@ -1290,6 +1292,10 @@ func (host *pythonLanguageHost) RuntimeOptionsPrompts(ctx context.Context,
 		}
 	}
 
+	// Note: Typechecker prompting has been removed to avoid automatically adding
```

Comment:
This note feels like it needs more context? We never had typechecker prompting so how's it removed?